tutorials.md

```diff
@@ -13,10 +13,10 @@ The tutorials assume that the reader has a basic knowledge of C, some C++, and L
 * [Installing MPICH2 on a single machine]({{ site.baseurl }}/tutorials/installing-mpich2/)
 * [Launching an Amazon EC2 MPI cluster]({{ site.baseurl }}/tutorials/launching-an-amazon-ec2-mpi-cluster/)
 * [Running an MPI cluster within a LAN]({{ site.baseurl }}/tutorials/running-an-mpi-cluster-within-a-lan/)
-* [Running an MPI hello world application]({{ site.baseurl }}/tutorials/mpi-hello-world/)
+* [Running an MPI hello world application]({{ site.baseurl }}/tutorials/mpi-hello-world/) [中文版]({{ site.baseurl }}/tutorials/mpi-hello-world/zh_cn)
```

Comment:
Can this be consistently formatted with the earlier usage (either both in `[ ]` or neither).